megahack_installer.sh

```diff
@@ -64,20 +64,24 @@ steam_path=${steam_path%/}
 echo "Using Steam path: $steam_path"
 
 # find proton version
-if [ -d "${steam_path}/steamapps/common/Proton - Experimental" ]; then proton_dir="${steam_path}/steamapps/common/Proton - Experimental"; fi
-if [ -d "${steam_path}/steamapps/common/Proton 7.0" ]; then proton_dir="${steam_path}/steamapps/common/Proton 7.0"; fi
-if [ -d "${steam_path}/compatibilitytools.d/GE-Proton7-43" ]; then proton_dir="${steam_path}/compatibilitytools.d/GE-Proton7-43"; fi # preferred version; more stable
-
-if [ ! -d "${proton_dir}" ]; then
-   echo "You dont have Proton Experimental, Proton 7.0 or GE-Proton7-43 installed!"
-   echo "Please set Geometry Dash to use either one of those versions (GE preferred)"
-   echo "To do that, go to GD's Steam page, click \"Properties\" > \"Compatibility\", enable \"Force the use of a specific Steam Play compatibility tool\" and select Proton 7.0 or Proton Experimental!"
-   echo "Proton GE has to be installed manually, use Proton 7.0 or Experimental if you are unsure how to do that."
-   echo "You have to start Geometry Dash at least once after changing it for Steam to download the new Proton version."
-   exit 1
-fi
+config_file="${steam_path}/steamapps/compatdata/322170/config_info"
+proton_dir=$(sed -n '2p' "$config_file") # get second line of config file
```

Comment:
I don't really like how this gets a specific line - if something changes in Steam that could get messed up.